client/src/components/Form/Select.js

```diff
@@ -9,6 +9,8 @@ export default function Select({
 	options,
 	placeholder,
 	required,
+	onChange = () => {},
+	className,
```

Comment:
> 💡 This seems like a reasonable extension to the behaviour that's already provided, but then all of the styles duplicate what's already in `src/components/Form/Form.scss`. Did you consider extending _the `Form` itself_, instead? With that approach:
> 
> ```js
> 	<Form onChange={handleChange}>
> 		<FormControls.Select
> 			label="Filter Topic"
> 			name="topic"
> 			options={topics}
> 			placeholder="Select a topic"
> 		/>
> 	</Form>
> ```
> 
> gives you something consistent with what's already on the suggestions page, and you just have to suppress the button (perhaps the same way the reset button is already handled).
> 
> ![Screenshot 2024-02-01 at 00 26 01](https://private-user-images.githubusercontent.com/785939/301350862-a5446f3b-9a7b-4712-bde9-0e9d21481cd0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2ODM1ODQsIm5iZiI6MTcwODY4MzI4NCwicGF0aCI6Ii83ODU5MzkvMzAxMzUwODYyLWE1NDQ2ZjNiLTlhN2ItNDcxMi1iZGU5LTBlOWQyMTQ4MWNkMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyM1QxMDE0NDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03OWY4NTQ0Mzg2ZmUxYzM4MTE3MTdmNjcyMDk4NWMzMmU5MDU1ZDg4YjQ4M2UyNWRmOWQ4N2M2MDdmMDI3MjQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.N9_ydV6QMi6sGEsbbm8ArfFaWYgeddlFlZLgK55uO2k)

i am wondering why we attach button to  select menu in case we might want to reuse it some other places like the one i did in resource list. wouldn't it be better to have separate button component rather than have it bind with select menu than we can use it anywhere without having to suppress  button every time we want to use select menu i understand this might me out side of scope of this ticket but as a suggestion. please correct me if i am wrong.

Reply:
> > I think we need to be able to use the select menu on its own without having to wrap it around a form.
> 
> In that case a third option would be to have a separate select for use _outside_ a form, leaving this one as the select for use _inside_ a form (the "namespacing" of `FormControls.Select` was intended to indicate this).
> 
> What made me think about making _the form_ configurable was that it keeps all of the styling and our handling of controls more consistent.

Or we can render buttons only where they are needed, rather than within the Form and have Form component  to provide a reusable form structure. in that case we don't need reintroduce new select.